@xen-orchestra/web-core/lib/components/ui/toaster/UiToaster.vue

```diff
@@ -0,0 +1,103 @@
+<!-- v3 -->
+<template>
+  <div :class="toVariants({ accent })" class="ui-toaster">
+    <div class="label">
+      <VtsIcon class="information-icon" :accent :icon="faCircle" :overlay-icon="icon" />
+      <div class="content">
+        <div class="typo h5-semi-bold">
+          <slot />
+        </div>
+        <div v-if="slots.description" class="description typo p2-regular">
+          <slot name="description" />
+        </div>
+      </div>
+      <UiButtonIcon class="close-icon" :icon="faXmark" :accent size="medium" />
+    </div>
+    <div v-if="slots.actions" class="actions">
+      <slot name="actions" />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import UiButtonIcon from '@core/components/ui/button-icon/UiButtonIcon.vue'
+import { toVariants } from '@core/utils/to-variants.util'
+import type { IconDefinition } from '@fortawesome/fontawesome-common-types'
+import { faXmark, faCheck, faCircle, faInfo, faExclamation } from '@fortawesome/free-solid-svg-icons'
+import { computed } from 'vue'
+
+type ToasterAccent = 'info' | 'success' | 'warning' | 'danger'
+
+const props = defineProps<{
+  accent: ToasterAccent
+}>()
+
+const slots = defineSlots<{
+  default(): any
+  description?(): any
+  actions?(): any
+}>()
+
+const states: Record<ToasterAccent, { icon: IconDefinition; color: ToasterAccent }> = {
+  info: { icon: faInfo, color: 'info' },
+  success: { icon: faCheck, color: 'success' },
+  warning: { icon: faExclamation, color: 'warning' },
+  danger: { icon: faXmark, color: 'danger' },
+}
+
+const icon = computed(() => states[props.accent].icon)
+</script>
+
+<style scoped lang="postcss">
+.ui-toaster {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 1.6rem;
+  border: 0.1rem solid;
+  border-radius: 0.4rem;
+  gap: 0.8rem;
+
+  .label {
+    display: flex;
+
+    .content {
+      margin-inline: 1.6rem 0.8rem;
+    }
+
+    .information-icon {
+      margin-bottom: auto;
+      font-size: 2.7rem;
+    }
+
+    .close-icon {
+      margin-left: auto;
+    }
+  }
+
+  .actions {
+    margin-inline-start: auto;
+  }
```

Comment:
Since you're using `display: flex` , I think it's better to use all of its power, rather than using `margin ` with auto. What do you think ? and according to DS, we can remove `margin-inline` and use `gap`  in  `.label`
```suggestion
  .label {
    display: flex;
    align-items: flex-start;
    justify-content: space-between;
    gap: 1.6rem;

    .content {
      flex-grow: 1;
    }

    .information-icon {
      font-size: 2.7rem;
    }

    .close-icon {
      flex-shrink: 0;
    }
  }

  .actions {
    align-self: flex-end;
  }
```

Reply:
The gap set 1.6 rem to each side, but we have to set 0.8rem on right and 1.6rem on the left side.

Reply:
After testing some things, here is my version:
```html
<!-- v3 -->
<template>
  <div :class="toVariants({ accent })" class="ui-toaster">
    <div class="content">
      <VtsIcon class="information-icon" :accent :icon="faCircle" :overlay-icon="icon" />
      <div>
        <div class="typo h5-semi-bold">
          <slot />
        </div>
        <div v-if="slots.description" class="typo p2-regular">
          <slot name="description" />
        </div>
      </div>
      <UiButtonIcon class="close-icon" :icon="faXmark" :accent size="medium" />
    </div>
    <div v-if="slots.actions" class="actions">
      <slot name="actions" />
    </div>
  </div>
</template>

<style scoped lang="postcss">
.ui-toaster {
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  padding: 1.6rem;
  border: 0.1rem solid;
  border-radius: 0.4rem;
  gap: 0.8rem;

  .content {
    display: flex;
    align-items: flex-start;
    gap: 1.6rem;

    .information-icon {
      font-size: 2.7rem;
    }

    .close-icon {
      margin-inline-start: auto;
      flex-shrink: 0;
    }
  }

  .actions {
    align-self: flex-end;
  }

  &.accent--info {
    background-color: var(--color-info-background-selected);
    border-color: var(--color-info-item-base);
  }

  &.accent--success {
    background-color: var(--color-success-background-selected);
    border-color: var(--color-success-item-base);
  }

  &.accent--warning {
    background-color: var(--color-warning-background-selected);
    border-color: var(--color-warning-item-base);
  }

  &.accent--danger {
    background-color: var(--color-danger-background-selected);
    border-color: var(--color-danger-item-base);
  }
}
</style>

```
I've kept some of my suggested changes, and some of the changes proposed by @J0ris-K.
I've removed some CSS classes, so the `content` one have been moved to the parent element. The name still not 100% convinces me, maybe `first-line` is more appropriate, I don't know.